app/assets/stylesheets/new_board/_key_frames.scss

```diff
@@ -0,0 +1,8 @@
+@keyframes heightAnimation {
```

Comment:
This name sounds kinda generic, can we make it more clear about what it does exactly? Also, the `98%` value reminded me of this post https://dev.to/maxart2501/percentages-in-css-you-re-using-them-wrong-kpn, maybe this can  be improved as well using a different unity?